app/helpers/glossary_helper.rb

```diff
@@ -76,14 +76,23 @@ def replace_glossary_terms(doc, sample, map, pattern)
   end
 
   def create_glossary_link(doc, term, content)
+    toggle = "dropdown-#{doc.search("//a[class='glossary-term']").count}"
```

Comment:
This doesn't seem to work. `#replace_glossary_terms` creates all the links before inserting any of them into the document, so the count here will always be 0. Can we instead generate a random id with something like `SecureRandom.hex(8)`? That would also guard against repeated ids on pages where we're replacing glossary terms in multiple regions (for example on lesson pages, `doc` can in one instance refer to the learning objectives, and in another the lesson body).